tests/sentry/utils/test_committers.py

```diff
@@ -15,3 +20,42 @@ def test_tokenize_path():
     assert list(tokenize_path('foo/bar')) == ['bar', 'foo']
     assert list(tokenize_path('foo\\bar')) == ['bar', 'foo']
     assert list(tokenize_path('foo.bar')) == ['foo.bar']
+
+
+class GetPreviousReleasesTestCase(TestCase):
```

Comment:
i just flipped the order in this test (and moved it) to reflect the tweaked code